src/basics.md

```diff
@@ -128,26 +128,16 @@ fn main() {
     let mut rng = rand::thread_rng();
     
     // Integers are uniformly distributed over the type's whole range:
-    let n1: u32 = rng.gen();
+    let n1: u8 = rng.gen();
     let n2: u16 = rng.gen();
-    let n3: u8 = rng.gen();
-    println!("Random u32, u16, u8: {}, {}, {}", n1, n2, n3);
+    println!("Random u8: {}; random u16: {}", n1, n2);
     println!("Random i32: {}", rng.gen::<i32>());
```

Comment:
I would suggest for the formatting to be uniform for all types. each on separate line